docs/release-notes/pytorch2.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**New Features**
+
+-  Environments: Add experimental PyTorch2.0 images.
+
+   -  We are providing a set of images with `pytorch==2.0.1`, `python==3.10.12`, and `CUDA==11.8`.
+      These images have not been fully tested against certain integrations, and are experimental.
```

Comment:
:orphan:

**New Features**

-  Environments: Add the following PyTorch2.0 images (experimental): ``pytorch==2.0.1``, ``python==3.10.12``, and ``CUDA==11.8``.

Reply:
@dzhu please review my proposed edit